consensus/interface.go

```diff
@@ -97,3 +97,11 @@ type HeadersPoolSubscriber interface {
 	RegisterHandler(handler func(headerHandler data.HeaderHandler, headerHash []byte))
 	IsInterfaceNil() bool
 }
+
+// ConsensusRating defines the behaivour of a component able to handle/monitor the rating of nodes which are
+// participating in consensus
+type ConsensusRating interface {
```

Comment:
PeerQualityHandler?

Reply:
PeerHonestyHandler ?

Reply:
it should not contain the name Rating at all. In order to not cause misunderstanding.

Reply:
It would be PeerHonestyHandler